librus/__init__.py

```diff
@@ -39,11 +39,11 @@ def list_exams(self):
         # TODO we should be iterating explicitly over links to calendar items' details; doing unstructured "grepping" for now
         for element in response.html.search_all("szczegoly/{}'"):
             details = self._html_session.get(url=f"https://synergia.librus.pl/terminarz/szczegoly/{element[0]}")
-            yield self._parse_exam(details.html.find('table.decorated.small'))
+            yield self._parse_exam(details.html.find('table.decorated.medium'))
```

Comment:
is this change intentional?

Reply:
yes, afaik they changed the class of the table to "decorated medium center"
"table.decorated.small" didn't work for me